packages/psss-server/src/routes/LogoutHandler.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { UnauthenticatedRouteHandler } from './UnauthenticatedRouteHandler';
+
+export class LogoutHandler extends UnauthenticatedRouteHandler {
+  async buildResponse() {
+    this.req.sessionCookie?.reset();
```

Comment:
Oooh love the use of this `sessionCookie?.reset()`. I keep forgetting this functionality exists now 😂